redisinsight/ui/src/packages/redisearch/src/index.html

```diff
@@ -6,8 +6,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>RediSearch plugin</title>
   <script type="module" src="./main.tsx"></script>
-  <link rel="stylesheet" href="../../../../../../vendor/global_styles.css" />
-  <link rel="stylesheet" href="../../../../../../vendor/dark_theme.css" />
+  <!-- <link rel="stylesheet" href="../../../../../../vendor/global_styles.css" />
+  <link rel="stylesheet" href="../../../../../../vendor/dark_theme.css" /> -->
```

Comment:
the same. Are you sure that this is not a mistake and styles will still work?

Reply:
sure
index.html needs only for developing the plugin. The builded plugin doesn't contain this file.